src/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -62,6 +62,7 @@ See the LICENSE file in the project root for more information.
       <NativeLibrary Include="uuid.lib" />
       <NativeLibrary Include="bcrypt.lib" />
       <NativeLibrary Include="normaliz.lib" />
+      <NativeLibrary Include="Crypt32.lib" />
```

Comment:
```suggestion
      <NativeLibrary Include="crypt32.lib" />
```

Reply:
Crypt32 is the casing in the SDK, but so is Uuid.lib a couple lines above. But I doubt anyone sane would enable NTFS case sensitivity on random directories in the SDK, so it's moot.